examples/simple/src/users/UserEdit.tsx

```diff
@@ -45,6 +45,7 @@ const EditActions = () => (
 const UserEditForm = () => {
     const { permissions } = usePermissions();
     const { save } = useSaveContext();
+    if (!save) return null;
```

Comment:
I'm surprised that `save` may be undefined, isn't this an error in the context definition?

Reply:
Because the `SaveContext` is optional. You can use a `Form` without it by providing your own `onSubmit`